src/v2/controllers/FormController.ts

```diff
@@ -243,7 +244,15 @@ export default Controller.extend({
       sessionStorageHelper.removeStateHandle();
 
       const currentViewState = this.options.appState.getCurrentViewState();
-      Util.redirectWithFormGet(currentViewState.href);
+
+      // OKTA-635926: add user gesture for android ov enrollment
+      const ovEnrollment = window.location.href.includes('redirect_uri=https%3A%2F%2Flogin.okta.com');
```

Comment:
Adding this check so the button does not appear in normal login flows and only OV enrollment login flows